app/controllers/questions_controller.rb

```diff
@@ -21,6 +23,7 @@ def create
     @question = Question.create question_params.merge(user: current_user)
 
     if @question.errors.empty?
+      PrivatePub.publish_to "/questions", question: @question.to_json
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.